app/presenters/ohms_transcript_display.rb

```diff
@@ -43,6 +43,27 @@ def display
     content_tag("div", safe_join(paragraph_html_arr), class: "ohms-transcript-container")
   end
 
+  def test_render
+    render "works/ohms_footnote_reference"
+  end
+
+  # The HTML for the inline tooltip and footnote reference.
+  # We use a template for this; even with dozens of footnotes,
+  # it doesn't appear to slow down the page load significantly.
+  def footnote_html(number)
+    # OhmsXmlValidator is in charge of validating the footnotes at
+    # upload time and rejecting ones that are invalid.
+    # We can thus assume the footnotes are valid at display time;
+    # if the array of footnotes doesn't contain a footnote for `number`,
+    # we are *not* going to throw an error.
+    raw_footnote = model.footnote_array[number.to_i - 1]  || ''
+    if raw_footnote == ''
+      Rails.logger.warn("WARNING: Reference to empty or missing footnote #{number} for OHMS transcript #{model.accession}")
+    end
+    render "works/ohms_footnote_reference",
+      footnote_text: raw_footnote.gsub('"', '&quot;'),
```

Comment:
I'm suspicious of this gsub -- if we want to escape for HTML, we should use a proper existing HTML escaping routine, not try to re-create one that only escapes quotes.

Reply:
should be a test around this -- create a test with that input has HTML special characters that aren't meant to be HTML tags, test teh output is it what you expect?

Reply:
Done!